app/views/spree/admin/adjustments/edit.html.haml

```diff
@@ -1,4 +1,5 @@
-= render :partial => 'spree/admin/shared/order_tabs', :locals => { :current => 'Adjustments' }
+= render :partial => 'spree/admin/shared/order_page_title', locals: { current: 'Adjustments' }
```

Comment:
current is a variable used only in order_tabs not in the new page_title so you shouldnt send it to page title partial, this is enough:
```suggestion
= render :partial => 'spree/admin/shared/order_page_title'
```

this applies to all calls to order_page_title

Reply:
change made to page title to remove current variable

Reply:
you may have had trouble with git as it looks like some changes were reverted to previous state, the current variable is now back here.
Always have a look at https://github.com/openfoodfoundation/openfoodnetwork/pull/5482/files before review :+1: